src/main/java/core/basesyntax/fruit/dto/FruitDtoParserImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.fruit.dto;
+
+public class FruitDtoParserImpl implements FruitDtoParser {
+    private static final String IS_NUMERIC = "[0-9]+";
+    private static final String LINE_SPLITERATOR = ",";
+    private static final String FRUITS = "banana,apple";
```

Comment:
let's not bind to banana and apple, we can have a lot of different fruits
let's remove this check and assume that everything that comes from file is the name of a fruit